composer.json

```diff
@@ -28,39 +28,37 @@
   "minimum-stability": "dev",
   "homepage": "http://www.taotesting.com",
   "require": {
-    "oat-sa/generis": "12.5.2",
-    "oat-sa/tao-core": "39.6.0",
-    "oat-sa/extension-tao-community": "7.0.1",
-    "oat-sa/extension-tao-funcacl": "5.3.2",
-    "oat-sa/extension-tao-dac-simple": "5.1.0",
-    "oat-sa/extension-tao-testtaker": "7.2.1",
-    "oat-sa/extension-tao-group": "6.2.0",
-    "oat-sa/extension-tao-item": "10.0.1",
-    "oat-sa/extension-tao-itemhtml": "5.1.0",
-    "oat-sa/extension-tao-itemqti": "22.0.6",
-    "oat-sa/extension-tao-itemqti-pci": "6.1.0",
-    "oat-sa/extension-tao-itemqti-pic": "5.3.3",
-    "oat-sa/extension-tao-test": "13.4.2",
-    "oat-sa/extension-tao-testlinear": "3.2.2",
-    "oat-sa/extension-tao-testqti": "35.2.1",
-    "oat-sa/extension-tao-outcome": "10.1.1",
-    "oat-sa/extension-tao-outcomeui": "8.0.1",
-    "oat-sa/extension-tao-outcomerds": "6.2.0",
-    "oat-sa/extension-tao-outcomekeyvalue": "5.3.1",
-    "oat-sa/extension-tao-outcomelti": "3.1.8",
-    "oat-sa/extension-tao-delivery": "14.5.0",
-    "oat-sa/extension-tao-delivery-rdf": "9.0.1",
-    "oat-sa/extension-tao-lti": "11.1.1",
-    "oat-sa/extension-tao-ltideliveryprovider": "10.1.1",
-    "oat-sa/extension-tao-revision": "7.0.0",
-    "oat-sa/extension-tao-mediamanager": "9.0.0",
-    "oat-sa/extension-pcisample": "2.5.0",
-    "oat-sa/extension-tao-backoffice": "4.1.0",
-    "oat-sa/extension-tao-proctoring": "18.1.0",
-    "oat-sa/extension-tao-clientdiag": "7.3.1",
-    "oat-sa/extension-tao-eventlog": "2.5.0",
-    "oat-sa/extension-tao-task-queue": "5.0.0",
-    "oat-sa/extension-tao-testqti-previewer": "2.8.2"
+    "oat-sa/generis": "~12.5.3",
+    "oat-sa/tao-core": "~40.1.1",
+    "oat-sa/extension-tao-community": "~7.0.1",
+    "oat-sa/extension-tao-funcacl": "~5.3.2",
+    "oat-sa/extension-tao-dac-simple": "~5.1.0",
+    "oat-sa/extension-tao-testtaker": "~7.2.1",
+    "oat-sa/extension-tao-group": "~6.2.0",
+    "oat-sa/extension-tao-item": "~10.0.1",
+    "oat-sa/extension-tao-itemqti": "~22.1.0",
+    "oat-sa/extension-tao-itemqti-pci": "~6.1.1",
+    "oat-sa/extension-tao-itemqti-pic": "~5.3.3",
+    "oat-sa/extension-tao-test": "~13.4.2",
+    "oat-sa/extension-tao-testqti": "~35.2.2",
+    "oat-sa/extension-tao-outcome": "~10.2.1",
+    "oat-sa/extension-tao-outcomeui": "~8.2.1",
```

Comment:
I know that it's auto-generated, but, please, remove `~` from version constraint of extensions

Reply:
done :)